### PR TITLE
Creates QA PUT requests

### PR DESCRIPTION
**Pull request by griseldasanchez**
Creates QA PUT requests:
- report question
- mark question as helpful
- report answer
- mark answer as helpful

### CODE REVIEW

File: client/src/components/QA/QACardQuestions.jsx [hunk 1]
```diff
@@ -54,8 +65,14 @@ const QACardQuestions = ({ questionsInfo, productName }) => {
           <div key={index}>
             <div className="qa-card-sample" data-testid="qa-card-sample" key={index}>
               <b><div className="qa-div">Q: {question.question_body}
-                <div className="qa-helpfulness-right" data-testid="qa-helpfulness-right">Helpful? <a href='' className='right-spacing'>Yes ({question.question_helpfulness}) </a>
-                  <QAAddAnswerModal question={question} index={index} productName={productName} />
+              <div className="qa-helpfulness-right righ-spacing" data-testid="qa-helpfulness-right">
```
Comment (NylonEric) [inline client/src/components/QA/QACardQuestions.jsx:68]: "right" spelling

File: server/atelier-queries.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const getProductReviewsMeta = (productId, headers) => {
 
 // Returns the questions as an array
 const getProductQuestions = (productId, headers) => {
-  return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${productId}&count=5000`, headers)
+  return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${productId}&count=500`, headers)
```
Comment (NylonEric) [inline server/atelier-queries.js:50]: Good idea limiting the get results. I wonder what a good amount should be?

File: client/src/components/QA/QACardAnswers.jsx [hunk 3]
```diff
@@ -32,6 +30,18 @@ const QACardAnswers = ({ currentAnswers }) => {
     setToggle(true);
   }, []);
 
+  const APICall = (answerId) => {
```
Comment (NylonEric) [inline client/src/components/QA/QACardAnswers.jsx:33]: These function names could probably be more specific to your module/api endpoint.

File: server/atelier-queries.js [hunk 4]
```diff
@@ -70,14 +70,65 @@ const postAnswer = (body, headers) => {
...
+    .then(result => {
+      return result;
+    })
+    .catch(err => {
+      throw err;
+    });
+}
+
+// Reports a question
+const reportQuestion = (questionId, headers) => {
```
Comment (NylonEric) [inline server/atelier-queries.js:94]: These names are super clear.

**Review by NylonEric — APPROVED** [review]
This all looks good. I mentioned a couple function names could be clarified since there are quite a few API calls to differentiate.

File: client/src/components/QA/QACardAnswers.jsx (hunk 3 above)
Comment (griseldasanchez) [reply client/src/components/QA/QACardAnswers.jsx:33]: Will update these

**Pull request merged**